src/core/execution/SpawnExecution.ts

```
@@ -86,7 +90,24 @@ export class SpawnExecution implements Execution {
     return true;
   }
 
-  private getSpawn(center?: TileRef): Spawn | undefined {
+  private getSpawn(center?: TileRef, player?: Player): Spawn | undefined {
+    if (player !== undefined) {
+      if (this.mg.config().startingOwnedTiles(player) !== undefined) {
+        let tiles = this.mg.config().startingOwnedTiles(player);
+        if (tiles == undefined || !tiles.length) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)